packages/app/components/trending/trending.web.tsx

```diff
@@ -65,56 +71,38 @@ export const Trending = () => {
   const contentWidth = useContentWidth();
   const bottomBarHeight = usePlatformBottomHeight();
   const isMdWidth = contentWidth >= breakpoints["md"];
-  const numColumns =
-    contentWidth <= breakpoints["md"]
-      ? 3
-      : contentWidth >= breakpoints["lg"]
-      ? 3
-      : 2;
-  const [days, setDays] = useParam("days", { initial: "1" });
+  const [filter, setFilter] = useParam("filter", { initial: "music" });
+  const contextValues = useMemo(
+    () => ({ filter, setFilter }),
+    [filter, setFilter]
+  );
 
   const { data: list, isLoading } = useTrendingNFTS({
-    days: Number(days),
+    filter: filter,
   });
-  const chuckList = useMemo(() => {
-    return chunk(list, numColumns);
-  }, [list, numColumns]);
+
   const keyExtractor = useCallback(
-    (_item: NFT[], index: number) => `${index}`,
+    (_item: NFT, index: number) => `${index}`,
     []
   );
 
   const renderItem = useCallback(
-    ({
-      item: chuckItem,
-      index: itemIndex,
-    }: ListRenderItemInfo<NFT[] & { loading?: boolean }>) => {
+    ({ item, index }: ListRenderItemInfo<NFT & { loading?: boolean }>) => {
       return (
-        <View
-          tw="mx-auto mb-px flex-row space-x-px px-0 md:space-x-6 md:px-6 lg:space-x-8 lg:px-4 xl:px-0"
-          style={{ maxWidth: contentWidth }}
-        >
-          {chuckItem.map((item, chuckItemIndex) => (
-            <Card
-              key={item.nft_id}
-              nft={item}
-              numColumns={numColumns}
-              href={`${getNFTSlug(item)}?initialScrollIndex=${
-                itemIndex * numColumns + chuckItemIndex
-              }&days=${days}&type=trendingNFTs`}
-            />
-          ))}
-          {chuckItem.length < numColumns &&
-            new Array(numColumns - chuckItem.length)
-              .fill(0)
-              .map((_, itemIndex) => (
-                <View key={itemIndex.toString()} tw="flex-1" />
-              ))}
+        <View tw="mx-auto max-w-screen-xl flex-row md:px-4">
+          <ListCard
+            nft={item}
+            showClaimButton
+            href={`/${getNFTSlug(
```

Comment:
`getNFTSlug` adds the `slash` so we can remove it from here.

Reply:
fixed